src/reducers/index.ts

```diff
@@ -1,16 +1,13 @@
 import { combineReducers } from 'redux';
-
 import namespaces from './Namespaces';
-import serviceGraphFilterState from './ServiceGraphFilterState';
 import { KialiAppState } from '../store/Store';
 import messageCenter from './MessageCenter';
-import serviceGraphDataState from './ServiceGraphDataState';
+import ServiceGraphReducer from './ServiceGraph';
```

Comment:
should these names start with lowercase now? @mtho11  I seem to remember you changing alot of these names to be all lowercase as their first character (??)